docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -116,6 +118,7 @@ variable length ``VARBINARY`` data type. There is no way to create a
 Phoenix table in Trino that uses the ``BINARY`` data type, as Trino
 does not have an equivalent type.
 
+.. include:: jdbc-type-mapping.fragment
```

Comment:
the "jdbc-type-mapping" documents things other than JDBC type mapping (eg `case-insensitive-name-matching` for _name_ mapping, or `metadata.cache-ttl` for metadata caching).

as such, it is not suitable for inclusion within "Type mapping" section

Reply:
Never mind, @hashhar appears to be resolving this in https://github.com/trinodb/trino/pull/8643 where he split out the non-type mapping parameters into a different fragment.